pybind_instructions.md

```diff
@@ -0,0 +1,82 @@
+# Porting an algorithm from C++ to Python
+
+## Table of Contents
+- [Why?](#why)
+- [How it works](#how-it-works)
+- [Requirements](#requirements)
+
+## Why?
+The goal of this project is to allow users to take an algorithm written in C++ and translate it into runnable Python code.
+This could allow a user to develop C++ code and run it on the EmotiBit's firmware, software, and through Python, hitting three different metrics.
```

Comment:
How about "this allows user to rapidly prototype and test algorithms in python, while maintaining the confidence that the performance is going to be exactly the same on firmware and software"? or something like that.